include/gdf/cffi/types.h

```diff
@@ -20,86 +24,136 @@ typedef enum {
     GDF_INT64,
     GDF_FLOAT32,
     GDF_FLOAT64,
-    GDF_DATE32,   	/**< int32_t days since the UNIX epoch */
-    GDF_DATE64,   	/**< int64_t milliseconds since the UNIX epoch */
-    GDF_TIMESTAMP,	/**< Exact timestamp encoded with int64 since UNIX epoch (Default unit millisecond) */
+    GDF_DATE32,   	/// int32_t days since the UNIX epoch
```

Comment:
Putting a comment on the same line, as in for a data member or enum requires `<`.

https://www.stack.nl/~dimitri/doxygen/manual/docblocks.html

Reply:
Ok, I'll fix it. The Eclipse Doxygen plugin accepts just `///` without the `<` as a marker of a same-line comment.